x-pack/plugins/security_solution/cypress/objects/rule.ts

```diff
@@ -215,7 +215,7 @@ export const machineLearningRule: MachineLearningRule = {
 };
 
 export const eqlRule: CustomRule = {
-  customQuery: 'process where process_name == "explorer.exe"',
+  customQuery: 'any where process.name == "which"',
```

Comment:
This seems indicative of a data issue and it would be nice to test the event.category functionality, but this is a great improvement!